terraform/modules/cloudwatch/main.tf

```diff
@@ -0,0 +1,49 @@
+resource "aws_cloudwatch_event_rule" "ecs_scheduled_event" {
+  name        = local.cloudwatch_event_rule_name
+  description = "Scheduled event rule to run ECS task"
+  schedule_expression = var.cron_expression
+}
+
+resource "aws_cloudwatch_event_target" "ecs_event_target" {
```

Comment:
for the event target we should be setting this as a conditional step, right now we only have ecs as the target but in the future this could change. I think there should be some variable "target_type" that will control whether this block gets run. if the target type is ecs you would run this, something like:

for_each = var.target_type == "ecs" ? 1 : 0